packages/components/src/core/Button/index.stories.tsx

```diff
@@ -225,41 +223,40 @@ export const LivePreview = {
       type,
       icon,
       disabled,
-      disabledIndex,
     }: {
       sdsStyle: (typeof SDS_STYLES)[number];
       type: (typeof SDS_TYPES)[number];
       icon: (typeof ICON_OPTIONS)[number];
       disabled: (typeof DISABLED_OPTIONS)[number];
-      disabledIndex: number;
     }) {
       return (
-        <div style={disabledLevel}>
-          <h6 style={disabledLabel}>
-            Disabled: <b>{disabled ? "true" : "false"}</b>
-          </h6>
+        <div style={penultimateLevel}>
           {PSEUDO_STATES.map((state) => {
             return (
-              <div style={stateLevel}>
-                {disabledIndex % 2 ? (
-                  false
-                ) : (
-                  <h6 style={stateLabel}>
-                    State: <b>{state}</b>
-                  </h6>
+              <div style={bottomLevel}>
+                {(disabled === false ||
```

Comment:
@liaprins maybe add a comment to explain the purpose of the condition!

Reply:
Removed irrelevant `disabled` iterations: when combined with all pseudo-states except default, `hover=false` is impossible